lib/yarv/opt_aset.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module YARV
+  # ### Summary
+  #
+  # `opt_aset` is an instruction for `recv[obj] = set` format # TODO expand me
+  #
+  # ### TracePoint
+  #
+  # TODO
```

Comment:
@kddnewton Could you please help me in identifying tracepoints per operation. I tried 

```ruby

irb(main):009:0>  RubyVM::InstructionSequence.compile("{}[:key]='val'", nil,nil,nil,peephole_optimization: false).trace_points
=> [[1, :line]]```
But it is not clear which yarv operation is causing tracepoint from this output

Reply:
Thanks, it makes sense now